jenkins/jobs/kata-containers-crio-PR/config.xml

```diff
@@ -119,29 +128,32 @@ pr_number=&quot;${PULL_NUMBER}&quot;
 pr_branch=&quot;PR_${pr_number}&quot;
 
 export ghprbGhRepository=&quot;${REPO_OWNER}/${REPO_NAME}&quot;
-
-echo &quot;Env Variables:&quot; 
-env
-
-echo &quot;This Job will test CRI-O changes using Kata Containers runtime.&quot;
-echo &quot;Testing PR number ${pr_number}.&quot;
+export GOROOT=&quot;/usr/local/go&quot;
+export PATH=${GOPATH}/bin:/usr/local/go/bin:/usr/sbin:/sbin:${PATH}
 
 # Put our go area into the Jenkins job WORKSPACE tree
 export GOPATH=${WORKSPACE}/go
 mkdir -p &quot;${GOPATH}&quot;
 
-# Export all environment variables needed.
-export GOROOT=&quot;/usr/local/go&quot;
-export PATH=${GOPATH}/bin:/usr/local/go/bin:/usr/sbin:/sbin:${PATH}
-
 # CRI-O repository
 crio_repo=&quot;github.com/kubernetes-incubator/cri-o&quot;
 # Kata Containers Tests repository
 tests_repo=&quot;github.com/kata-containers/tests&quot;
 
-# Clone repositories
+echo &quot;This Job will test CRI-O changes using Kata Containers runtime.&quot;
+echo &quot;Testing PR number ${pr_number}.&quot;
+
+tests_repo_dir=&quot;${GOPATH}/src/${tests_repo}&quot;
+
+# Clone the tests repository
+mkdir -p $(dirname &quot;${tests_repo_dir}&quot;)
+[ -d &quot;${tests_repo_dir}&quot; ] || git clone &quot;https://${tests_repo}.git&quot; &quot;${tests_repo_dir}&quot;
+
+# Install Go
+${GOPATH}/src/${tests_repo}/.ci/install_go.sh -p
```

Comment:
I'll see if I can steal some of this for the metric CI setup as well - presently that has GO installed in the VM image so it can `go get` some repos before doing the build/run - using `install_go.sh` looks like a better long term solution.